### PR TITLE
Draft: Untie FPS cap from client tick / input rate

### PR DESCRIPTION
**Pull request by x2048**
This PR Makes the game run smoother when playing at capped FPS which could be useful to save battery on mobile devices allowing client to tick more frequently than drawing the frames.

## To do

This PR is a Work in Progress.

- [ ] Fix the FPS meter and FPS graph (F5)
- [ ] Add setting to control client tick rate (hardcoded at 60 at the moment)

## How to test

1. Set max FPS to 5
2. Start any game
3. Try to do small moves - it should be possible now

### CODE REVIEW

File: src/client/game.cpp [hunk 1]
```diff
@@ -1193,8 +1194,8 @@ void Game::run()
 
 		// Calculate dtime =
 		//    m_rendering_engine->run() from this iteration
-		//  + Sleep time until the wanted FPS are reached
-		draw_times.limit(device, &dtime);
+		//  + Sleep time until the wanted tick rate are reached
+		draw_times.limit(device, &dtime, 60);
```
Comment (lhofhansl) [inline src/client/game.cpp:1198]: Why the hardcoded 60?

Comment (x2048) [reply src/client/game.cpp:1198]: Make this a setting is in the TODO list for the PR.

Comment (x2048) [reply src/client/game.cpp:1198]: And 60 is the default FPS cap in MT

**Comment by AbduSharif** [comment]
Works great gor thr brief testing I done.

**Comment by AbduSharif** [comment]
I say that but then I stumble at least across one error, I dug up a dirt block but it took a few seconds for it to get dug and got this error:
```
ERROR[MeshUpdate]: Waiting for texture default_dirt.png^default_grass_side.png^[crack:1:0 timed out.
```

My Minetest settings incase they're relevant:
```
fsaa = 2
leaves_style = fancy
enable_3d_clouds = true
smooth_lighting = true
screen_w = 2960
enable_shaders = true
fps_max = 5
screen_h = 1440
maintab_LAST = local
menu_last_game = minetest
fps_max_unfocused = 5
video_driver = ogles2
enable_mesh_cache = true
meshgen_block_cache_size = 250
mesh_generation_interval = 5
selected_world_path = /storage/emulated/0/Android/data/net.minetest.minetest_tickratefix/files/Minetest/worlds/Tickrate Test
enable_damage = false
server_announce = false
mainmenu_last_selected_world = 1
creative_mode = true
```

**Comment by AbduSharif** [comment]
Also the hotbar selection doesn't seem to work at all.

**Pull request closed without merging**